app/models/ride.rb

```diff
@@ -1,3 +1,9 @@
 class Ride < ApplicationRecord
   belongs_to :amusement_park
+  has_many :mechanic_rides
+  has_many :mechanics, through: :mechanic_rides
+
+  def mechanic_to_ride 
+    mechanics.distinct.pluck(:name)
+  end
```

Comment:
This method does not implement the US properly. 

The US asks for you to return all of the unique mechanics that are working across all of a park's rides. This method gets a unique list of mechanics that are working on one ride and only one ride. This method should be on the `AmusementPark` model, not here.